myuw/templates/base.html

```diff
@@ -225,6 +243,24 @@ <h2 class="sr-only" id="app_messages_mobile_label">MyUW Announcements</h2>
                 <!-- end message banner -->
                 {% endblock %}
 
+                {% if display_onboard_message %}
+		{% block mobile_tour_banner %}
+		<!-- Tour message banner (mobile only)-->
+		<div class="myuw-banner-mobile app-messages" id="tour_messages_mobile" aria-labelledby="app_messages_mobile_label" role="complementary">
+		    <div class="message-banner" style="margin-top:-17px;">
+		        <h2 class="sr-only" id="app_messages_mobile_label">MyUW Announcements</h2>
```

Comment:
`id="app_messages_mobile_label"` This one should also be changed, since this is the ID used for the app messages above when we create one using the admin tool. 

Perhaps something related to the ID of the banner. so something like `id="tour_messages_mobile_label"`

You'll want to edit the the `aria-labelledby="app_messages_mobile_label"` above on line 249 so that the aria-labelledby attribute matches whatever ID you choose.

Reply:
Good catch. ID is changed.